index.js

```diff
@@ -2,6 +2,21 @@ const TestRunner = require('./src/TestRunner')
 
 exports.runTests = async function (options) {
   return new Promise((resolve, reject) => {
+    // Look for the config file. If one isn't found, keep
+    // going up the directory tree until you're at root
+    while (!process.cwd === '/') {
```

Comment:
I don't think that changing the working directory is safe. Especially when you do not change the directory back to its original directory when completed. I think really what you should be doing is something like:

- Start at `__dirname` of the caller. I think that `runTests` should take a `dirname` option in case it's being directly used via API instead of CLI. If `dirname` is not a provided option, it should default to `process.cwd()` as it does now.
- Use `path.dirname(currentPath)` to get the parent path. 
- While true, loop until current path is undefined.